spring-cloud-netflix-sidecar/src/test/java/org/springframework/cloud/netflix/sidecar/SidecarApplicationTests.java

```diff
@@ -27,22 +27,60 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.junit.Assert.assertThat;
 
-@RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(classes = SidecarApplication.class, webEnvironment = WebEnvironment.RANDOM_PORT, value = {
-		"spring.application.name=mytest", "spring.cloud.client.hostname=mhhost",
-		"spring.application.instance_id=1", "eureka.instance.hostname=mhhost",
-		"sidecar.port=7000" })
 public class SidecarApplicationTests {
 
-	@Autowired
-	EurekaInstanceConfigBean config;
+	@RunWith(SpringJUnit4ClassRunner.class)
+	@SpringBootTest(classes = SidecarApplication.class, webEnvironment = WebEnvironment.RANDOM_PORT, value = {
+			"spring.application.name=mytest", "spring.cloud.client.hostname=mhhost",
```

Comment:
After removed it used for `instanceId` generation if `eureka.instance.hostname` is not present so `NewPropertyEurekaTestConfigBeanTest`

```
Expected: "mhhost:mytest:1"
     but: was "127.0.0.1:mytest:1"
Expected :mhhost:mytest:1
Actual   :127.0.0.1:mytest:1
```

Thus I can't change assertion to match regex (because ip can be different depending on environment, in my case I have an alias for `lo0`) or simply rollback?